.github/workflows/pr-build-workflow.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     services:
       rabbitmq:
-        image: rabbitmq
+        image: rabbitmq:management
         ports:
           - 5672:5672
```

Comment:
Don't we need to map the management port (15762)?

Reply:
I just copy/paste all of this stuff blindly 😄 
Is that just enough to have another entry like `- 15762:15762` ?
Do we have some tests for that to be sure that they run properly?

Reply:
Hmmm - now I don't know how the previous PR builds were good; for example `RabbitAdminIntegrationTests` has `@RabbitAvailable(management = true)`

But we have other tests (e.g. `EnableRabbitIntegrationTests`) that don't check for the management plugin is available, but require it.

Reply:
Oh; it has a try/catch

```java
	@Test
	public void deadLetterOnDefaultExchange() {
		this.rabbitTemplate.convertAndSend("amqp656", "foo");
		assertThat(this.rabbitTemplate.receiveAndConvert("amqp656dlq", 10000)).isEqualTo("foo");
		try {
			Client rabbitRestClient = new Client("http://localhost:15672/api/", "guest", "guest");
			QueueInfo amqp656 = rabbitRestClient.getQueue("/", "amqp656");
			if (amqp656 != null) {
				assertThat(amqp656.getArguments().get("test-empty")).isEqualTo("");
				assertThat(amqp656.getArguments().get("test-null")).isEqualTo("undefined");
			}
		}
		catch (Exception e) {
			// empty
		}
	}
```